### DISCUSSION TITLE
listen to store events

### DISCUSSION THREAD

**Discussion by jongbonga** [salesforce/akita#800] [Q&A] [answered]
I want to change the `loading` default value of my entity stores. 
The current default is `true`, but I want it to be `false`.

If there is a way to do it globally (through all/certain stores) great, else I want to listen to when `resetStores()` is triggered so I can `setLoading` to `false`, because resetting the store set my entities loading states to `true`

**Comment by NetanelBasal** [comment, marked as answer]
No. You should do it per store.

No, you can wrap it in a function and do what you need.
